code/modules/projectiles/guns/energy/kinetic_accelerator.dm

```diff
@@ -29,21 +31,23 @@
 /obj/item/gun/energy/kinetic_accelerator/examine(mob/user)
 	. = ..()
 	if(in_range(user, src))
-		if(max_mod_capacity)
+		if(modable)
```

Comment:
This variable is completely unnecessary. Just use `if(max_mod_capacity)` like it was used here.

Reply:
Fair point. I figured making a modable variable, rather than max_mod_capacity being zero would work better, but in the end it is an unnecessary variable when max being zero does the same. One moment.